changelog.json

```diff
@@ -1,5 +1,6 @@
 {
   "releases": {
+    "2.8.2-beta3": [],
```

Comment:
There is nothing user-facing included in this beta, just the metrics 😅 

Thoughts? @billygriffin @tidy-dev